spec/services/builders/external_projects_spec.rb

```diff
@@ -19,12 +19,13 @@
   end
 
   context 'when there is no external project' do
+    let(:repo_name) { 'ruby_repo' }
+    let(:full_name) { "github/#{repo_name}" }
+
     let(:repository_data) do
       {
         id: 247_546_341,
-        name: 'ruby_repo',
-        full_name: 'github/ruby_repo',
-        description: 'this is a uby repo'
+        name: full_name
       }
```

Comment:
I know it probably seems like an overkill, but why not use the factory `gitub_api_client_pull_requests_events_payload`? It would be more precise to me.